firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -2115,6 +2140,118 @@ extension TelemetryWrapper {
             }
             GleanMetrics.Pings.shared.fxSuggest.submit()
 
+        // MARK: - Toolbar
```

Comment:
Uh, you're gonna hate me for this, but, the new method we're looking to implement, telemetry wise, going forward, is creating specific telemetry utilities for anything that uses telemetry in Redux. So when you instantiate your middleware, or your reducer or whatnot, you'll have a telemetry utility for that thing, where you'll do the needful, and thus avoiding embiggening the `TelemetryWrapper`. An example is [MicrosurveyTelemety](https://github.com/mozilla-mobile/firefox-ios/blob/912eae5754061554f7fe94283c7433edf9878b1b/firefox-ios/Client/Frontend/Microsurvey/MicrosurveyTelemetry.swift)

anything that's currently not easy to go through the utility, we can still add in the wrapper, but we'll (hopefully very soon) be getting rid of the convoluted headache that it is.